README.md

```diff
@@ -12,7 +12,13 @@
 $ sh ./gradlew -Pjdk18_home=<jdk18_home_dir> -Pllvm_home=<libclang_dir> clean verify
```

Comment:
I refered to `libclang_dir`, however I wonder if the value in the angle brackets has to be renamed as `llvm_home` as well.

Reply:
This has been updated in a recent PR - please try to refresh your branch